src/components/checkbox/checkbox.tsx

```diff
@@ -134,4 +135,14 @@ export class Checkbox {
         this.change.emit(this.mdcCheckbox.checked);
         this.modified = true;
     }
+
+    @Watch('indeterminate')
+    private onIndeterminateChanged(newValue) {
```

Comment:
If you set this to protected, eslint will stop complaining.

Reply:
I noticed we used protected everywhere else but didn't know it was because of eslint. Good catch, I'll update it!